src/worker.py

```
@@ -871,6 +1065,250 @@ async def api_login(req, env):
+        return ok(None, _GENERIC_OK)
+
+    # Per-account cooldown: silently drop if a token was issued in the last 5 minutes.
+    try:
+        recent = await env.DB.prepare(
+            "SELECT id FROM email_verification_tokens"
+            " WHERE user_id=? AND created_at > datetime('now', '-5 minutes')"
+        ).bind(user_row.id).first()
+        if recent:
+            return ok(None, _GENERIC_OK)
+    except Exception:
+        pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)